src/PreferencesDialog.ui

```diff
@@ -20,7 +20,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```

Comment:
This is a common mistake while using Qt Creator. It saves the current tab page according to what you have when you press save. You have to remember to save while being in the first tab, or undo this change afterwards, before the commit.

Reply:
@mgrojo - your not on gitter so I'll ask here and related to above.
How do you feel about disconnecting from Qt Creator?
My impression is that the IDE is starting to _get in the way_ WRT code/future changes.

Reply:
Thanks for showing - have to keep this in mind!